### PR TITLE
Update selectors and add/edit js steps

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/P000449.yaml [hunk 1]
```diff
@@ -8,118 +8,120 @@ contact_form:
         - selector: input#field_ABC03F4E-8760-440F-BA41-22107A2F7BCF
```
Comment (tomdionesotes) [inline members/P000449.yaml]: looks like the form url redirects to https://www.portman.senate.gov/meet/contact fyi

File: members/xCA_L_Fong.yaml [hunk 2]
```diff
@@ -39,6 +39,11 @@ contact_form:
           options:
             max_length: 2000
           required: true
+    - javascript:
```
Comment (tomdionesotes) [inline members/xCA_L_Fong.yaml]: any blacklists to remove here?

Comment (OdeclasV) [reply members/xCA_L_Fong.yaml]: oh yeah, missed those. One sec

File: members/P000449.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/P000449.yaml]: good catch. Updated

**Pull request merged**